wpilibj/src/main/java/edu/wpi/first/wpilibj/shuffleboard/ContainerHelper.java

```diff
@@ -0,0 +1,88 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.shuffleboard;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.Sendable;
+
+/**
+ * A helper class for Shuffleboard containers to handle common child operations.
+ */
+final class ContainerHelper {
+
```

Comment:
Why is wpiformat not catching this blank line @calcmogul?

Reply:
That's a bug. I reproduced it locally, and it has to do with improper handling of comments (the comment contains the class keyword).

Reply:
oh... lol